.github/workflows/versions.yml

```diff
@@ -0,0 +1,368 @@
+name: versions
+on:
+  - pull_request
+  - push
+
+jobs:
+  verify-tags:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Verify Setup For Tags
+        run: |
+          echo "Verify 0.0.2:"
+          curl https://raw.githubusercontent.com/bpkg/bpkg/0.0.2/setup.sh | bash
+          bpkg --version | grep 0.0.2 >/dev/null || {
```

Comment:
Something is nagging at me that this line has issues but I can't seem to get the problem in clear focus.

It might come to me later...